src/main/resources/public/js/index.js

```diff
@@ -0,0 +1,70 @@
+    var roundScore = 0;
+
+    /**
+     * Función que obtiene una pregunta
+     * @param {number} questionID es el numero de pregunta
+     */
+    function getQuestionByID(questionID) {
+        return new Promise(function(resolve, reject) {
+            $.get(`/question/get?idQuestion=${questionID}`, (questionJSON) => {
+                resolve(questionJSON);
+            });
+        });
+    }
+    /**
+     * Función que obtiene el número total de preguntas
+     */
+    function getQuestionCount() {
+        return new Promise(function(resolve, reject) {
+            $.get(`/question/count`, (count) => {
+                resolve(count);
+            });
+        });
+    }
+
+    /**
+     * Función que inicializa los puntajes de los tableros
+     */
+    function setInitialScore2Boards(){
+        $("#center_score").html(`<p>${roundScore}</p>`);
+        $("#left_score").html(`<p>${roundScore}</p>`);
+        $("#right_score").html(`<p>${roundScore}</p>`);
+    }
+    /**
+     * Función que reproduce los sonidos
+     * @param {string} audioPath es la ruta en la que se encuentra el audio por reproducir
+     */
+    function playSound(audioPath){
+        var sound = new Audio (audioPath);
+        sound.play();
+    }
+
+    /**
+     * Función que agrega clase hide que se encuentra en el css
+     * @param {string} element elemento al que se le quiere agregar clase hide
+     */
+    function addHideClass(element){
+        for (let i=1; i<6; i++){
+            $(`#${element}${i}`).addClass("hide");
+        }
+    }
+    /**
+     * Función que remueve clase hide que se encuentra en el css
```

Comment:
```suggestion
     * Remueve clase hide que se encuentra en el css
```
"Función que" no agrega nada a la documentación.